task/rhacs-deployment-check/3.71/rhacs-deployment-check.yaml

```diff
@@ -20,12 +20,8 @@ spec:
   params:
     - name: rox_central_endpoint
       type: string
-      description: Name of secret containing the address:port tuple for RHACS Stackrox Central.
-      default: rox-central-endpoint
-    - name: rox_api_token
-      type: string
-      description: Name of secret containing the RHACS StackRox API token with CI permissions.
-      default: rox-api-token
+      description: The address:port tuple for RHACS Stackrox Central.
+      default: central.stackrox.svc:443
     - name: rox_image
       description: Image providing the roxctl tool.
       default: quay.io/stackrox-io/roxctl:3.71.0
```

Comment:
nit: let's maybe bump this to a latter version? Also, WDYT about adding all the PRs that we've introduced within the last few weeks to the 4.4.2 milestone so that folks can get started using M2M commands sooner?

Reply:
I'd be in favour of backporting the roxctl changes to 4.4.2, assuming they make it in time.